package ru.itmo.tgbot.exception.user

class ParticipationInEventNotFoundException: RuntimeException()